python/lsst/pipe/tasks/sspAuxiliaryFile.py

```diff
@@ -0,0 +1,44 @@
+from lsst.daf.butler import FormatterV2
+from lsst.resources import ResourcePath
+from io import BytesIO
+from typing import Any
+
+__all__ = ["SSPAuxiliaryFile", "SSPAuxiliaryFileFormatter"]
+
+
+class SSPAuxiliaryFile():
+    """Class to hold information about auxiliary files needed for
+    solar system object ephemeris calculations.
+    """
+    fileContents = None
+
+    def __init__(self, fileContents):
+        self.fileContents = fileContents
```

Comment:
I am not 100% reading through the code. I can't tell if you are using binary as a passthrough here and the data is actually stored in a readable way, or if you are indeed writing straight binary to disk. If it is the later we generally discourage that. Is this actually just passthrough to a csv or something?